brainiak/funcalign/srm.py

```diff
@@ -132,6 +144,8 @@ class SRM(BaseEstimator, TransformerMixin):
     rho2_ : array, shape=[subjects]
         The estimated noise variance :math:`\\rho_i^2` for each subject
 
+    comm_ : mpi4py.MPI.Intracomm
+        The MPI communicator containing the data
```

Comment:
`comm` should be used instead of `comm_` everywhere, because it is given as a parameter, not computed from data:
http://scikit-learn.org/stable/developers/contributing.html#instantiation
http://scikit-learn.org/stable/developers/contributing.html#estimated-attributes